extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/IgnoreJsonDeserializeClassBuildItem.java

```diff
@@ -10,13 +12,22 @@
  */
 public final class IgnoreJsonDeserializeClassBuildItem extends MultiBuildItem {
 
-    private final DotName dotName;
+    private final List<DotName> dotNames;
 
     public IgnoreJsonDeserializeClassBuildItem(DotName dotName) {
-        this.dotName = dotName;
+        this.dotNames = List.of(dotName);
+    }
+
+    public IgnoreJsonDeserializeClassBuildItem(List<DotName> dotNames) {
+        this.dotNames = dotNames;
     }
 
+    @Deprecated
```

Comment:
Maybe add a javadoc why this is deprecated? Also perhaps mark for removal.
```suggestion
    @Deprecated(forRemoval= true)
```